config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/ingredients", to: "ingredients#index"
+  get "/recipes/update", to: "recipes#update"
```

Comment:
See note in view about this path and controller routing -- consider POST to a RecipeIngredientsController